Makefile

```diff
@@ -59,6 +59,14 @@ container:
     --build-arg REPO_INFO=${REPO_INFO} \
     -t ${IMAGE}:${TAG} .
 
+.PHONY: railgun-container
+railgun-container:
+	docker build \
+		-f Dockerfile.railgun \
+    --build-arg TAG=${TAG} --build-arg COMMIT=${COMMIT} \
+    --build-arg REPO_INFO=${REPO_INFO} \
+    -t ${IMAGE}:${TAG} .
+
```

Comment:
I'm a bit uncomfortable with starting to transport railgun code into the base repo without covering the context with several levels of supporting documentation and otherwise communicating what this is to readers.

My preference between this and the above `Dockerfile` would be to pull back and continue confining our changes to `railgun/` for now, what are your thoughts on that?

Reply:
Resolving as duplicate of https://github.com/Kong/kubernetes-ingress-controller/pull/1175#discussion_r612727905 to keep the conversation in one place. Like I described there, I'm coming from an assumption that we should integrate railgun's build process with KIC's early.